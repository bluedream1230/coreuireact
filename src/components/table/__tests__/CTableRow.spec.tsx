import * as React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import { CTableRow } from '../../../index'

test('loads and displays CTableRow component', async () => {
  const { container } = render(<CTableRow/>)
  expect(container).toMatchSnapshot()
})

test('CTableRow customize', async () => {
    const { container } = render(
        <CTableRow
            className="bazinga"
            active={true}
            align="middle"
            color="info"
        >
            Test
        </CTableRow>
    )
    expect(container).toMatchSnapshot()
    expect(container.firstChild).toHaveClass('align-middle')
    expect(container.firstChild).toHaveClass('table-active')
    expect(container.firstChild).toHaveClass('table-info')
    expect(container.firstChild).toHaveClass('bazinga')
    expect(container.firstChild).toHaveTextContent('Test')
})