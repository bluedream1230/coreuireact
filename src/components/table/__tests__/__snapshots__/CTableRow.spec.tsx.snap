// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CTableRow customize 1`] = `
<div>
  <tr
    class="align-middle table-active table-info bazinga"
  >
    Test
  </tr>
</div>
`;

exports[`loads and displays CTableRow component 1`] = `
<div>
  <tr />
</div>
`;
